internal/resolvers/twitch/user_loader.go

```diff
@@ -48,17 +59,36 @@ func (l *UserLoader) Load(ctx context.Context, login string, r *http.Request) (*
 		return nil, cache.NoSpecialDur, resolver.ErrDontHandle
 	}
 
+	streamResponse, err := l.helixAPI.GetStreams(&helix.StreamsParams{UserLogins: []string{login}})
+	if err != nil {
```

Comment:
Let's handle `err != nil` and `len(streamResponse.Data.Streams) != 1` in the same way: Output it as a userResponse instead